iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/mempool_introspection.inl

```diff
@@ -30,9 +30,8 @@ MemPoolIntrospection<MemoryManager, SegmentManager, SenderPort>::MemPoolIntrospe
 {
     m_senderPort.activate(); // corresponds to offer
 
-    /// @todo create a wrapper function which takes care of the 16 character limitation of the thread name
-    // set thread name
-    pthread_setname_np(m_thread.native_handle(), "MemPoolIntr");
+    cxx::string<16> threadName{"MemPoolIntr"};
+    pthread_setname_np(m_thread.native_handle(), threadName.c_str());
```

Comment:
This is not really what the `todo` suggested. Now we have a magic number distributed over several files. Could you maybe wrap the `pthread_setname_np` function in the utils?

Reply:
I chose to implement it in a free function as the first step. Once we create our own pthread class, this will become a member method.